Tella/Utils/Utils.swift

```diff
@@ -37,3 +37,12 @@ extension View {
         AnyView(self)
     }
 }
+
+extension View {
+    func customCardStyle() -> some View {
```

Comment:
I think it's better to create something like 'CardFrameView' and add the padding to the content  or use  'CardFrameView' and adapt it to be used in all the app 
    
    
    
```
 struct CardFrameView<Content:View>: View {
  
        var content : () -> Content
        
        init(@ViewBuilder content : @escaping () -> Content) {
            self.content = content
        }
        
        var body: some View {
            VStack(spacing: 0) {
                self.content()
                    .padding(.all, 16)
            }.background(Color.white.opacity(0.08))
                .cornerRadius(15)
                .padding(.top, 12)
        }
    }
```